daemon/src/state.rs

```diff
@@ -355,17 +355,30 @@ where
 /// Get the user found at `urn`.
 ///
 /// # Errors
+///   * Storage access fails
+pub async fn get_user<S>(peer: &Peer<S>, urn: Urn) -> Result<Option<Person>, Error>
+where
+    S: Clone + Signer,
+{
+    peer.using_storage(move |store| identities::person::get(store, &urn))
+        .await?
+        .map_err(Error::from)
+}
+
+/// Get the local user found at `urn`.
 ///
-///   * Resolving the user fails.
-///   * Could not successfully acquire a lock to the API.
-pub async fn get_user<S>(peer: &Peer<S>, urn: Urn) -> Result<Option<LocalIdentity>, Error>
+/// # Errors
+///   * Storage access fails
+pub async fn get_local<S>(peer: &Peer<S>, urn: Urn) -> Result<Option<LocalIdentity>, Error>
```

Comment:
While we're changing this stuff, is there a reason this should take a `Urn`? We could just use `local::default` to get the `LocalIdentity` associated with the store right?

Reply:
You can have 0..n local identities